app/src/main/res/layout/dashboard_spotlight_concat.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
you aren't using any databinding functionality in this layout, so it doesn't need to be a databinding layout